spec/models/job_spec.rb

```diff
@@ -1,5 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe Job, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context "#instantiate" do
+    it "should generate new instance" do
+      newclass=described_class.new
+      expect(newclass).to be_a(described_class)
```

Comment:
You should use Job instead of described_class, it's a little more descriptive about the subject. 

```ruby
expect(job).to be_a(Job)
```